onnxscript/optimizer/_constant_folding.py

```diff
@@ -696,6 +703,14 @@ def process_node(self, node: ir.Node):
         if any(x is None for x in input_values):
             return None
 
+        if any(input.size > self._input_size_limit for input in input_values):
+            if logger.isEnabledFor(logging.DEBUG):
+                input_sizes = [input.size for input in input_values]
+                logger.debug(
+                    f"Skipping constant folding for op {node.op_type} due to large input size: {input_sizes}"
```

Comment:
Use %s for logs

Reply:
Does that still apply when the code is already protected by `isEnabledFor` ?

Reply:
IIUC, the main value is in avoiding unnecessary formatting. If we are enabled for it, then nothing is achieved by delaying the formatting.

Reply:
That's true. Just easier to check for than having to make an exception for the lint rule I suppose?